### DISCUSSION TITLE
Data Visualisation

### DISCUSSION THREAD

**Discussion by troyhunt** [HaveIBeenPwned/ux-rebuild#6] [Ideas]
This is a broad idea, but in essence it comes down to how information about breaches and search results is down. Maybe a good starting point is to look at how the ABC did the visualisation of HIBP data last year: https://www.abc.net.au/news/2023-05-18/data-breaches-your-identity-interactive/102175688

I'm not necessarily saying that's the best approach, but particularly as the data has grown I reckon there are much better ways of visualising things than just a flat table of results. This may not replace that, but rather complement it. Same with domain searches.

I'd love to see some ideas about how we could make the data itself more interesting and provide people with with something both more useful and more engaging to look at. I'm especially interested in any examples you're able to point to, there must be some great visualisations of large data sets out there.

**Comment by neilcaldwell** [comment]
PwnedWebsites could be sorted by date, with newest at top, with options to sort by size/alpha/date, perhaps a filter for type of data [email/IP/names/passwords/phone/etc]

**Comment by troyhunt** [comment]
Love the look of stuff like this, what data should HIBP turn into visualisations?

<img width="1276" alt="image" src="https://github.com/HaveIBeenPwned/ux-rebuild/assets/273244/da72e91a-e85e-4cc0-bce7-947449084489">

**Comment by cmlh** [comment]
These are [part of] the visualised link analysis I produce with [Maltego](https://github.com/cmlh/Maltego-haveibeenpwned):

![image](https://github.com/HaveIBeenPwned/ux-rebuild/assets/136826/d605aa3a-33a3-4163-b2d2-4b9bd91f27e4)

![image](https://github.com/HaveIBeenPwned/ux-rebuild/assets/136826/1e0cd11e-a815-4fdd-96b1-55d2f5a44518)

**Comment by yousefebrahimi0** [comment]
1. The current numeric display of breach statistics could be improved with simple visualizations that show scale and trends. Considering adding small trend charts or indicators for showing the history of the data leak about each company ... can be a good idea, I think.
Some ideas: https://informationisbeautiful.net/visualizations/worlds-biggest-data-breaches-hacks/

2. What do you think about publishing monthly pdf report of the data breaches? like: https://www.verizon.com/business/resources/T70f/infographics/2024-dbir-public-sector-snapshot.pdf

**Comment by troyhunt** [comment]
The info is beautiful site has been a fav for many years now 😊

We could do something similar on a dedicated page that visualises the HIBP data, but I'm struggling to see how that would work on the home page. Open to ideas though.

Anything report related (like the DBIR) can be a huge amount of work, we'd need it to be automatically driven off the data we have. Again, ideas welcome!
